src/main/java/teammates/ui/webapi/CreateAccountAction.java

```diff
@@ -112,7 +112,7 @@ private String importDemoData(String instructorEmail, String instructorName)
             }
 
             count -= rand.nextInt(numResponses / numIntervals);
-            timestamp -= 120; // 2 minutes interval
+            timestamp -= (30 * 1000); // 2 minutes interval
```

Comment:
Sin of having comment... If you are to change the value to mean something, please change the comment alongside... Although I should have made a named variable for this number which is now a magic number.